README.md

```diff
@@ -50,6 +51,7 @@ parameters:
 		reportUnusedCatchesOfUncheckedExceptions: false
 		reportCheckedThrowsInGlobalScope: false
 		ignoreDescriptiveUncheckedExceptions: false
+		allowImplementationUnusedThrows: false
```

Comment:
`reportUnusedThrowsInImplementation` looks more clear, WDYT? And `false` as a default value

Reply:
in fact, it was the former name :)